src/main/java/com/backbase/ct/bbfuel/configurator/ContentServicesConfigurator.java

```diff
@@ -0,0 +1,37 @@
+package com.backbase.ct.bbfuel.configurator;
+
+import static com.backbase.ct.bbfuel.data.ContentServicesDataGenerator.createRepositories;
+
+import com.backbase.ct.bbfuel.client.common.LoginRestClient;
+import com.backbase.ct.bbfuel.client.contentservices.ContentServicesPresentationRestClient;
+import java.io.File;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class ContentServicesConfigurator {
+
+    private final ContentServicesPresentationRestClient contentServicesPresentationRestClient;
+    private final LoginRestClient loginRestClient;
+
+    @SuppressWarnings("squid:S1075")
+    public void ingestContentForPayments() {
+        
+        final String repositoryId = "payments";
+        final String targetPath = "/templates";
+        final File paymentTemplate = new File("src/main/resources/data/content/paymentTemplate.hbs");
```

Comment:
Does this work when building bb fuel jar and running it from somewhere that is not the maven project base dir?